client/jetpack-connect/woo-dna-config.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Internal dependencies
+ */
+import { isEnabled } from 'config';
+
+export default isEnabled( 'jetpack/connect/woo-dna' )
+	? {
+			'woocommerce-payments': {
+				name: ( translate ) => translate( 'WooCommerce Payments' ),
```

Comment:
I agree. However, this will change before release (sorry for this last-minute change, can be done in a different PR). The plugin name and help URL will come as part of the URL (the logic to "inject them" in the URL will be in the WPCom server), so there's no point in refactoring this.

Reply:
No, no issue, I'll open the PR (and the corresponding diff in wpcom) this evening or tomorrow.